msg/PoseBounds.msg

```diff
@@ -0,0 +1,11 @@
+# Bounds for named poses, e.g. to avoid spawning with other object overlap, or parking in a spot that is too small.
```

Comment:
So this is a bounding-box of free space at a given point? It feels a bit backward. From a user perspective I'd expect to 'ask' the simulator if my entity fits at a given pose and not what are the named pose and their available space...

Again this feels too specific imho.

Reply:
I think that asking whether something fits is not the most productive approach as a binary answer doesn't really inform how to continue.

Bounds are an optional field included in messages which are not regularly called (e.g. with high frequency, often). Not much burden to handle as you can simply ignore the field if you don't care about potential consequences of overlaps.

If you think about spawn points, charging points, loading points, all these can benefit from bounds information, so I don't believe there is anything backward about it.

Reply:
What would the actual use look like in a scenario where the user does care about overlaps?

```
-> named_pose = GetNamedPoses().poses[x]
-> user_defined_overlaps_computation(my_entity, named_pose.bounds)
```
??

Reply:
I made some changes to this that I believe address some of your concerns, coming soon.